test/persist_banking_pgsql.js

```diff
@@ -1182,6 +1203,20 @@ describe('Banking from pgsql Example', function () {
         });
     });
 
+    it('can save / retrieve a document to remote store', function(done) {
+        Customer.getFromPersistWithId(sam._id).then(function(samRemoteDoc) {
+            samRemoteDoc.bankingDocument = 'meow!';
+            return samRemoteDoc.persistSave();
+        }).then(function () {
+            return Customer.getFromPersistWithId(sam._id);
+        }).then(function(customerOutput) {
+            expect(customerOutput.bankingDocument).to.equal('meow!');
+            done();
+        }).catch(function(e) {
+            done(e);
```

Comment:
Does this trigger a failure condition, or do we need to assert manually, or let it throw and don't catch

Reply:
I removed the catch so if anything does go wrong it won't catch the error => cause the test to fail